recipes/codacy-coverage/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "codacy-coverage" %}
+{% set version = "1.3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b94651934745c638a980ad8d67494077e60f71e19e29aad1c275b66e0a070cbc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
I think you need to specify an [entry point](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=meta.yaml#python-entry-points) here:

https://github.com/codacy/python-codacy-coverage/blob/master/setup.py#L65

Reply:
Done in line 16:
```
    entry_points:
	    - python-codacy-coverage = codacy:main
```